packages/loader/driver-utils/src/adapters/compression/documentServiceCompressionAdapter.ts

```diff
@@ -14,6 +14,12 @@ export class DocumentServiceCompressionAdapter extends DocumentServiceProxy {
 		private readonly _config: ICompressionStorageConfig,
 	) {
 		super(service);
+		// Back-compat Old driver
```

Comment:
any other adapters that we should care about?
How do we ensure we do not miss any (and have no events flowing)?
It's probably fine, but I wonder, should loader be able to verify that driver is of same or higher version, and if so - assert that ability to subscribe to events is provided? It's rather weak check, and a bit ugly, so not sure if we want to go that route..

Reply:
It does not seem like we have more adapters as the compilation would have failed, I also scanned the code a bit. 
I don't want to compare package version, as their format also keeps changing. We could also check that 
if (service instanceOf EventEmiter), then listen to event.  OR
we could expose some prop on IDocumentServicePolicies and check that and then subscribe to event.